apps/meteor/client/components/UserStatusMenu.tsx

```diff
@@ -39,9 +55,13 @@ const UserStatusMenu = ({ onChange, optionWidth = undefined, initialStatus = 'of
 		hide();
 	});
 
-	const ref = useRef();
+	const ref = useRef<HTMLElement>(null);
```

Comment:
why not?
```suggestion
	const ref = useRef<HTMLButtonElement>(null);
```

Reply:
I don't exactly remember 🙈 let me revise this :p